public_records_portal/views.py

```diff
@@ -666,11 +666,15 @@ def upload_document():
     for file in files:
         document_upload_errors = {}
         secure_fname = secure_filename(file.filename)
-        document_filepath = app.config["UPLOAD_PUBLIC_LOCAL_FOLDER"] + '/' + form['request_id'] + '/' + secure_fname
-        doc_id, filename, errors = upload_file(file, form['request_id'], 0x1)
-        if (os.path.exists(document_filepath)):
+        # Searches through public and private folders for any pre-existing documents
```

Comment:
Why don't we just search in the database to see if a document with the same name exists for the current request?